lib/media-ast-from-string.js

```diff
@@ -131,4 +132,11 @@ const andRegExp = new RegExp(`^${expressionRE}(?:${andRE}${spaceRE})$`, 'i');
 const spaceWrapRegExp = new RegExp(`^${noSpaceRE}${noExpressionRE}${noSpaceRE}$`);
 const mediaRegExp = new RegExp(`^(?:${modifierRE}${spaceRE})?(?:${typeRE}(?:${andRE}${spaceRE}${expressionRE})?|${expressionRE})$`, 'i');
 
+/**
+ * remove spaces after left and before right bracket if spaces exist
+ */
+function trimNearBrackets(string) {
+	return string.replace('( ', '(').replace(' )', ')')
```

Comment:
What if it will be two or more spaces?

Reply:
good point!
@alex-knyazev I guess we need trim all spaces

Reply:
@Semigradsky fixed, please check, thanks